cache/cache_flags.go

```diff
@@ -1,56 +1,56 @@
 package cache
 
-// Flags are used to enable/disable certain internal caches
-type Flags int
+// Types are used to enable/disable certain internal caches
+type Types int
```

Comment:
I'm not sure I agree with renaming this. why do you think it's suitable to rename it?

Reply:
old one is prob fine agreed